example/js/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
+{"program":{"fileNames":["../../../../../usr/local/lib/node_modules/typescript/lib/lib.d.ts","../../../../../usr/local/lib/node_modules/typescript/lib/lib.es5.d.ts","../../../../../usr/local/lib/node_modules/typescript/lib/lib.dom.d.ts","../../../../../usr/local/lib/node_modules/typescript/lib/lib.webworker.importscripts.d.ts","../../../../../usr/local/lib/node_modules/typescript/lib/lib.scripthost.d.ts","./api/ICU4XDataProvider.d.ts","./api/ICU4XFixedDecimal.d.ts","./api/ICU4XLocale.d.ts","./api/ICU4XFixedDecimalGroupingStrategy.d.ts","./api/ICU4XFixedDecimalSignDisplay.d.ts","./api/ICU4XFixedDecimalFormatOptions.d.ts","./api/ICU4XFixedDecimalFormatResult.d.ts","./api/ICU4XFixedDecimalFormat.d.ts","./api/diplomat-runtime.d.ts","./node_modules/ava/index.d.ts","./test/fixed-decimal-ts.ts","./node_modules/@types/normalize-package-data/index.d.ts"],"fileInfos":["2dc8c927c9c162a773c6bb3cdc4f3286c23f10eedc67414028f9cb5951610f60",{"version":"f5c28122bee592cfaf5c72ed7bcc47f453b79778ffa6e301f45d21a0970719d4","affectsGlobalScope":true},{"version":"3f149f903dd20dfeb7c80e228b659f0e436532de772469980dbd00702cc05cc1","affectsGlobalScope":true},{"version":"7fac8cb5fc820bc2a59ae11ef1c5b38d3832c6d0dfaec5acdb5569137d09a481","affectsGlobalScope":true},{"version":"097a57355ded99c68e6df1b738990448e0bf170e606707df5a7c0481ff2427cd","affectsGlobalScope":true},"4c789bf1d1a6a93415ad22e29e92b43015ab8cc350a1fcb80da2f983157fb1c8","537655f4c75580debe0b9f01121e7798a93eb1e7dfd5daef3b0c327cf19cead0","66e622e4b07a9e66b7244e0e9914e000a2fcb9022c0431df2edd6af9f7967353","525e4fe8830e068e4fb17b2050cc7b4fd976059fe26a6dcefc792b114557e757","05b1cbb9037e336ccbc94f6dd6c04c9cff2b26f8e53cf0907f341a3885749cd2","a1ce5b74c07d97da2d87155fc4474c54246c89c583ad983669b1754705ab23dc","b28fbb5707eb264535708e95e6ae40f0eaf497bfb0fe9046ece60aa140e11c26","a7804262ac1135bacb47365edc98bae10f6071c41e05434c339bc058747668ef","6f26b0df3f5f250cf242aeae2441170b2652525991e652540a2eb2f79e763369","0ceec7923fb2a6e540f73682c7acffe9124c9c3517a02eac27632844b8a61251",{"version":"ef7d7ce0c240d1d5e4be26e62a9dc4a10156989c2e1b690ac1a56ac0053ea061","signature":"8e609bb71c20b858c77f0e9f90bb1319db8477b13f9f965f1a1e18524bf50881"},"6fa0008bf91a4cc9c8963bace4bba0bd6865cbfa29c3e3ccc461155660fb113a"],"options":{"module":6},"fileIdsList":[[6,7,8,11,12],[9,10],[13],[6,7,8,11,13,15]],"referencedMap":[[13,1],[11,2],[12,3],[16,4]],"exportedModulesMap":[[13,1],[11,2],[12,3]],"semanticDiagnosticsPerFile":[6,7,13,11,12,9,10,8,14,17,15,16,1,3,2,5,4]},"version":"4.7.4"}
```

Comment:
Nit: I think you should add .tsbuildinfo to the gitignore, according to the following:

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html

> These .tsbuildinfo files can be safely deleted and don’t have any impact on our code at runtime - they’re purely used to make compilations faster. We can also name them anything that we want, and place them anywhere we want using the [tsBuildInfoFile](https://www.typescriptlang.org/tsconfig#tsBuildInfoFile) option.

You can delete it in your next PR.